### ISSUE TITLE
Batch Norm 

### ISSUE THREAD

**Issue by pvskand** [Ignotus/theano-flownet#3]
In the original flownet implementation there is batch normalization after every conv layer. In the`FloatNetC.py`, I don't find any `batch_norm` layer. Any specific reason?

**Comment by Ignotus** [comment]
Which FlowNet configuration are you talking about?  I couldn't find any batch normalization in their protobuf file.

**Comment by pvskand** [comment]
Okay so I was looking at NVIDIA's [Pytorch](https://github.com/NVIDIA/flownet2-pytorch/blob/master/networks/FlowNetC.py#L17) implementation of `Flownet 2.0`. In that they had used `batch_norm` so I assumed the case with `Flownet 1.0`, but it's not the case.
